### ISSUE TITLE
Add simple Alibaba Cloud example

### ISSUE THREAD

**Issue by sanyer** [ContainerSolutions/terraform-examples#44]

**Comment by melihc1** [comment]
Simple VPC example (or another simple and free resource) - https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/vpc
Scripting and CI/CD should also be implemented for Alibaba Cloud
For CI/CD an account would also be needed

**Comment by sanyer** [comment]
Hey @melihc1 ,
Is this a note for the future?

**Comment by melihc1** [comment]
@sanyer, yes it is for the future. We wanted to clarify what we expect and what's needed for adding an Alibaba Cloud example

**Comment by sanyer** [comment]
Oh, ok ok. Makes sense.
